tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -150,14 +150,14 @@ sub run {
     my ($self, $run_args) = @_;
 
     if (check_var('IS_MAINTENANCE', 1)) {
-        my %maintenance_vars = qesap_calculate_az_address_range(slot => get_var("WORKER_INSTANCE"));
+        my %maintenance_vars = qesap_calculate_az_address_range(slot => get_var("WORKER_ID"));
```

Comment:
Sorry I did not notice this before. Let's use `get_required_var()` instead of `get_var()`